.github/workflows/rust.yml

```diff
@@ -82,7 +82,7 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - uses: dtolnay/rust-toolchain@nightly
-    - uses: dtolnay/rust-toolchain@1.65.0 # MSRV
+    - uses: dtolnay/rust-toolchain@1.74.0 # MSRV
```

Comment:
One of our dev-dependencies needs 1.81. We can just raise the MSRV: since the cargo resolver now takes this into account it will no longer break people's builds.